controller/src/main/java/org/jboss/as/controller/operations/global/ReadResourceHandler.java

```diff
@@ -613,6 +625,13 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
             }
         }
 
+        private void getWarning(ModelNode value, OperationContext context) {
```

Comment:
Nitpick: `getWarning` suggests this method should return something. I would suggest something like `addResponseWarning` or `addWarning`. Those sound more appropriate to describe what this method does.